mlrun/serving/server.py

```diff
@@ -771,7 +769,31 @@ async def run(body):
         model_endpoint_uids=model_endpoint_uids,
     )
 
-    return responses
+    # log the results as artifacts
+    num_of_meps_in_the_graph = len(server.graph.model_endpoints_names)
+    artifact_path = None
+    if (
+        "/{{run.uid}}" not in context.artifact_path
```

Comment:
```suggestion
        "{{run.uid}}" not in context.artifact_path
```
The uid is not necessarily a folder name.